data/emulator.js

```diff
@@ -2245,18 +2248,20 @@ class EmulatorJS {
                     textBox2.value = "";
                     textBox1.value = "";
                     if (this.controls[i][k] && this.controls[i][k].value !== undefined) {
-                        textBox2.value = this.controls[i][k].value;
+                        textBox2.value = this.localization(String(this.controls[i][k].value));
                     }
-                    if (this.controls[i][k] && this.controls[i][k].value2 !== undefined) {
-                        textBox1.value = this.controls[i][k].value2;
+                    if (this.controls[i][k] && this.controls[i][k].value2 !== undefined && this.controls[i][k].value2 !== "") {
+                        let value2 = String(this.controls[i][k].value2).split(":");
```

Comment:
Can we call the tostring function instead of using the String constructor

Reply:
Yes